vms/evm/metrics/prometheus/interfaces.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package prometheus
+
+import "github.com/ava-labs/libevm/metrics"
+
+var _ Registry = metrics.Registry(nil)
+
+type Registry interface {
+	// Call the given function for each registered metric.
+	Each(func(string, any))
```

Comment:
Nit: while most people could probably guess `name`, it seems like `any` is probably not super descriptive.
```suggestion
	Each(func(name string, metric any))
```